<launch>
    <arg name="model_dir" default="$(find strands_head_orientation)/models/default" />
    <arg name="queue_size" default="5" />
    <arg name="camera_namespace" default="/head_xtion" />
    <arg name="upper_body_detections" default="/upper_body_detector/detections" />
    <arg name="head_ori" default="/head_orientation/head_ori" />
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>

    <node pkg="strands_head_orientation" type="strands_head_orientation" name="head_orientation" output="screen" respawn="true">
        <param name="model_dir" value="$(arg model_dir)" type="string"/>
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="upper_body_detections" value="$(arg upper_body_detections)" type="string"/>
        <param name="head_ori" value="$(arg head_ori)" type="string"/>
    </node>
</launch>
